src/ansible_dev_tools/resources/server/creator.py

```diff
@@ -182,11 +190,42 @@ def playbook(
             init_path=str(init_path),
             output=CreatorOutput(log_file=str(self.tmp_dir / "creator.log")),
             project=project,
-            scm_org=scm_org,
-            scm_project=scm_project,
+            namespace=scm_org,
+            collection_name=scm_project,
             subcommand="init",
         )
         Init(config).run()
         tar_file = self.tmp_dir / f"{scm_org}-{scm_project}.tar.gz"
         create_tar_file(init_path, tar_file)
         return tar_file
+
+    def playbook_v2(
```

Comment:
@shatakshiiii I think we can remove this function and refactor `playbook` function to incorporate everything from here. As these functions are more or less the same barring one or two lines

Reply:
Added a `playbook_v1` function because v1 API will only be accepting the old _scm_org_ and _scm_project_ parameters. And later we will pass on these params to `playbook` function after naming them as _namespace_ and _collection_name_ respectively.